src/Docker.DotNet/DockerClient.cs

```diff
@@ -200,12 +200,14 @@ internal DockerClient(DockerClientConfiguration configuration, Version requested
             CancellationToken token)
         {
             var response = await PrivateMakeRequestAsync(timeout, HttpCompletionOption.ResponseContentRead, method, path, queryString, headers, body, token).ConfigureAwait(false);
```

Comment:
We should likely have any calls thought the client to `PrivateMakeRequestAsync` dispose of the `HttpResponseMessage` correct?